scripts/controlnet.py

```diff
@@ -672,8 +672,8 @@ def postprocess(self, p, processed, is_img2img=False, *args):
             for detect_map, module in self.detected_map:
                 if module in ["canny", "mlsd", "scribble", "fake_scribble", "pidinet"]:
                     detect_map = 255-detect_map
-                processed.images.extend([detect_map])
-        
+                processed.images.extend([Image.fromarray(detect_map)])
```

Comment:
Note that this line resolves the `'numpy.ndarray' object has no attribute 'save'` error correctly for once. With this, we technically don't need anymore to detect whether we're in img2img batch mode or not. However, it will create a mess in the output directory if we just remove the code. I suggest to tackle this in another PR.